spacy_llm/tasks/noop.py

```diff
@@ -4,12 +4,14 @@
 
 from ..registry import registry
 
+NOOP_PROMPT = "Don't do anything."
```

Comment:
Variables not needed outside of the module (ignoring tests) should be prefixed with `_`.

Reply:
Done.